### ISSUE TITLE
Please clarify the type of data you can visualize

### ISSUE THREAD

**Issue by TylerL-uxai** [defaultnamehere/zzzzz#21]
Hello! Can you please include a screen shot in your github so we all know what kind of graphs you can create? Thanks!

**Comment by dimartiro** [comment]
I think that it's only a chart.

![image](https://cloud.githubusercontent.com/assets/659400/13203022/b4fec6f8-d88b-11e5-8f4d-1d3b7c37ba43.png)

**Comment by amit2103** [comment]
Shouldn't there be a way to know the name of the friend?

**Comment by pielco11** [comment]
Yes, there is. For every id you can check the corresponding page and extract the name from "<title id="pageTitle"> name here </title>" and add this information to the corresponding id

**Comment by abdallahhamdy** [comment]
Been running the fetcher for 4 hours so far, not a single incident of recording fbappstatus or messengerapp status, I think it's only logging the web presence, it's possible they changed the api in the past 2 days 
![screenshot from 2016-02-22 21 03 30](https://cloud.githubusercontent.com/assets/1306659/13229118/fb586420-d9a7-11e5-8de9-10ac6a3c3bdc.png)

**Comment by JacobValdemar** [comment]
For some of my friends it shows only activity for status and for others friends the other categories work.

**Comment by pielco11** [comment]
Me too, I see only activity.

**Comment by amit2103** [comment]
@pielco11 Corresponding page means the CSV page or the html page/. if its the HTML page do I need to see the browser developer panel?

**Comment by pielco11** [comment]
@amit2103 For corresponding page, I mean the html page of the profile. If the ID is "1234" than you go to facebook.com/profile.php?id=1234 and from the source code of the page you can extract the name.

**Comment by amit2103** [comment]
@pielco11  Ohh...many thanks..

**Comment by SteffeyDev** [comment]
I know this chat is a bit old, but http://findmyfbid.com is great for finding the id given someone's name.  If someone could integrate this into the web app so that we can just type the name to pull up the graph, that would be awesome.  I might get around to it at some point but web development is not my forte.

**Comment by defaultnamehere** [comment]
You can use the included "fpapi.py" to do the lookup that you're talking about =]
